newIDE/app/src/EventsSheet/EventsTree/style.css

```diff
@@ -6,11 +6,14 @@
 .gd-events-sheet .rst__moveHandle {
     width: 10px;
     background: #4ab0e4;
-    border: 1px solid #49AFE4;
     box-shadow: none;
     border-radius: 0;
 }
 
+.gd-events-sheet .rst__moveHandle:hover {
+    background-color: #767676!important;
```

Comment:
Colors should be in the theme only, and probably different for both themes?

Reply:
Fixed